src/main/main.c

```diff
@@ -83,15 +83,17 @@ enum {
 #endif
 };
 
+/** File descriptor handler struct */
+struct fhs {
+	int fd;              /**< File Descriptor                   */
```

Comment:
`int fd` is only used on windows ?

perhaps change the type of fd to `SOCKET` to make it clear that it is a windows type:

```
typedef UINT_PTR        SOCKET;
```

it will also let the compiler warn about wrong api usage etc.

Reply:
As i can see SOCKET is casted to `int` through `SOK_CAST` like here:
https://github.com/baresip/re/blob/master/src/tcp/tcp.c#L510

I can make `re->fhs[i].fd` WIN32 only, with some extra ifdefs